src/components/sound.js

```diff
@@ -58,6 +61,14 @@ module.exports.Component = registerComponent('sound', {
         sound.setRolloffFactor(data.rolloffFactor);
       }
       sound.setLoop(data.loop);
+      sound.setLoopStart(data.loopStart);
+
+      // With a loop start specified without a specified loop end, the end of the loop should be the end of the file
+      if(data.loopStart != 0 && data.loopEnd == 0){ 
+        sound.setLoopEnd(sound.buffer.duration);
+      }
+      else sound.setLoopEnd(data.loopEnd);
```

Comment:
```
else {
  sound.setLoopEnd(data.loopEnd);
}
```
we always use brackets for if statements

Reply:
I see, will change.